### PR TITLE
[HU] Rules update

### PR DESCRIPTION
**Pull request by Spkz**
### Description

Updates `Rules/hu.md` by adding the new `Community Rules/#2` rule. It also contains a bit of rewording for `Community Rules/#3` and fixing a supposedly oversight minor mistake.

### Status

Finished

### Changelog

- Add `Communit Rules/#2`
- Reword `Community Rules/#3` and add a missing 'k'
- Apply Kurokami's review

### CODE REVIEW

**Comment by Spkz** [comment]
@Shadowthegod whenever you are available, please review the canges

File: wiki/Rules/hu.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ Közösségi Szabályok
...
-2.  **Legyetek rendesek egymáshoz.** Zaklatásnak, vagy bármi egyéb ellenségeskedésnek nincs helye az osu! közösségében. Azért vagyunk itt, hogy karikákat kattintsunk, dobokat üssünk, gyümölcsöket szedjünk, vagy akár tizennyolc gombot nyomjunk le egyszerre, nem azért, hogy gyökerek legyünk egymáshoz.
-3.  **Ne legyél seggfej.** Ha bármikor felmerül a kérdés a fejedben, hogy vajon megszeged-e ezt a szabályt, valószínűleg igen.
-4.  **Mi egy korhatár nélküli közösség vagyunk.** Ez azt jelenti, hogy 18+/felnőtt tartalom, mint például droghasználat, vagy bármiféle szexuális jellegű téma nem elfogadott. Ritka kivételek előfordulhatnak, ahol ez illő lehet, de általánosan beszélve, kerüljük ezeket.
-5.  **Ahol a szabályok nem érvényesülnek, ott a józan ész igen.** A moderátorok megítélésük alapján bírálhatják ezt, ahol szerintük szükséges.
+2.  **Játssz tisztán.** Bármiféle segédprogram használata, ami bármi előnyben részesít másokkal szemben, nem elfogadott. Ebbe beletartozik például: makroprogramok, célzást segítő programok, időkeret módosítók és így tovább. Ha egy program segítséget nyújt a játékban, amit egyébként egyedül kéne végezned, az nem szabályos.
```
Comment (Shadowthegod) [inline wiki/Rules/hu.md]: "ami bármi előny biztosít másokkal szemben" sounds better imo.

"Ha egy program azt csinálja, amit egyébként neked kellene, az nem szabályos." Same here.

**Comment by Spkz** [comment]
Modified both, according to suggestions. If there's nothing else to fix, it is ready to merge

**Comment by TPGPL** [comment]
@Shadowthegod could you check this one again?

**Pull request merged**